pkg/config/defaults.go

```diff
@@ -104,6 +104,10 @@ var (
 	defaultWinRemovableDrives            = true
 	defaultMetricsMatcherConfig          = IncludeMetricsMap{}
 	defaultRegisterMaxRetryBoSecs        = 60
+	defaultNtpPool                       = []string{} // i.e: []string{"time.cloudflare.com"}
+	defaultNtpEnabled                    = true
```

Comment:
If we don't provide any pool, should we disable it by default?
```suggestion
	defaultNtpEnabled                    = false
```

Reply:
But then when we add the pool we would need to change it to true and we would be changing the default behaviour. I would leave it like this

Reply:
The issue I see on that is agents without a defined pool (or that just update their agent) will see the following error in their logs: https://github.com/newrelic/infrastructure-agent/pull/1432/files#diff-93e924c55f2a6b30f1940dfae7efcacaa909d9915a19a38887670f177cd1032fR41